osf/external/chronos.py

```diff
@@ -78,6 +78,7 @@ def serialize_manuscript(cls, journal_id, preprint, status=ChronosSubmissionStat
                 }
             ],
             'UNDERLYING_DATASET_URL': preprint.node.absolute_url if preprint.node else '',
+            'LICENSE': preprint.license.node_license.name.upper() if preprint.license.node_license.name != 'No license' else 'NL',
```

Comment:
Since not every preprint has a license, this will break if there's no license attached at all.